lb.tf

```diff
@@ -126,8 +126,8 @@ resource "exoscale_compute" "lb" {
     ],
     "runcmd" = [
       "while lsof -F p /var/lib/dpkg/lock 2>/dev/null; do echo \"Waiting for dpkg lock...\"; sleep 15; done",
-      "curl -Lo /tmp/ursula.deb ${trimsuffix(var.bootstrap_bucket, "/")}/ursula.deb",
-      "dpkg -i /tmp/ursula.deb",
+      "curl -Lo /tmp/floaty.deb https://github.com/vshn/floaty/releases/download/v1.0.0/floaty_linux_amd64.deb",
```

Comment:
I'm not sure if we want to use a pinned version of the release, or whether we should make the effort to always download the latest stable version (may require more or alternative GitHub API calls)